docs/reference/configuration.md

```diff
@@ -65,6 +61,13 @@ This reference uses `.` to denote the nesting of values.
     + **Note:** `endpoint` `aws_region` `aws_access_key_id` `aws_secret_access_key` are not required and used mainly for experimental purposes when working with DynamoDB with different AWS credentials.
       {: .note }
     + `database.dynamodb.health_check_interval` `(duration : 0s)` - Interval to run health check for the DynamoDB instance (won't run if equal to 0).
+  + `database.cosmosdb` - Configuration section when using `database.type="cosmosdb"`
+    + `database.cosmosdb.readwritekey` `(string : "")` - If specified, will 
```

Comment:
Note that if not set we use the default - which make this one optional

Reply:
Done.